app/services/omniauth_sign_in_service.rb

```diff
@@ -0,0 +1,9 @@
+class OmniauthSignInService
+  def find_or_create_user!(access_token:)
+    data = access_token.info
```

Comment:
Can we pass just the `info` attribute instead send the entire `access_token` object and get that attribute and storing in a variable?